doc/releases/changelog-0.36.0.md

```diff
@@ -237,6 +237,25 @@
 
 <h4>Work easily and efficiently with operators</h4>
 
+* This release completes the main phase of PennyLane's switchover to an updated approach for
+  handling arithmetic operations between operators. The new approach is now enabled by default and
+  is intended to realize a few objectives:
+
+  1. To make it as easy to work with PennyLane operators as it would be with pen and paper.
+  2. To improve the efficiency of operator arithmetic.
+
+  In many cases, this update should not break code. If issues do arise, check out the
+  [updated operator troubleshooting page](https://docs.pennylane.ai/en/stable/news/new_opmath.html)
+  and don't hesitate to reach out to us on the
+  [PennyLane discussion forum](https://discuss.pennylane.ai/). As a last resort the old behaviour
+  can be enabled by calling `qml.operation.disable_new_opmath()`, but this is not recommended
+  because support will not continue in future PennyLane versions.
+  [(#5269)](https://github.com/PennyLaneAI/pennylane/pull/5269)
```

Comment:
```suggestion
  can be enabled by calling `qml.operation.disable_new_opmath()`, but this is not recommended
  because support will not continue in future PennyLane versions (v0.36 and higher).
  [(#5269)](https://github.com/PennyLaneAI/pennylane/pull/5269)
```